### PR TITLE
Unification

### PR DESCRIPTION
**Pull request by gimre-xymcity**
1. Introduces `\mathvar` and `\mathname`
2. `\hf` to use operatorname for H() which is frequently used (`\h` was already taken #sadpanda)
3. some index changes / fixes
4. removal of any leftover `textit` in favor of emph

### CODE REVIEW

File: whitepaper/chapters/blockchain_sync_flowchart.tex [hunk 1]
```diff
@@ -1,3 +1,4 @@
+\index{synchronization}
 \index{block!synchronization}
```
Comment (Jaguar0625) [inline whitepaper/chapters/blockchain_sync_flowchart.tex:2]: is there reason for double index ?

File: whitepaper/chapters/blocks.tex [hunk 2]
```diff
@@ -67,12 +67,12 @@ \section{Blocks}
...
 
 \field{Height} is the block sequence number.
-The first block, called the \nind{nemesis block}, has a height of one.
+The first block, called the \nind{nemesis block}\index{block!nemesis}, has a height of one.
 Each successive block increments the height of the previous block by one.
 
 \field{Timestamp} is the number of milliseconds that have elapsed since the nemesis block.
 Each successive block must have a timestamp greater than that of the previous blocks because block time is strictly increasing.
-Each network tries to keep the average time between blocks close to \textit{target block time}.
+Each network tries to keep the average time between blocks close to \nind{target block time}.
```
Comment (Jaguar0625) [inline whitepaper/chapters/blocks.tex:75]: i wonder if we should replace all \emph with \nind ?

File: whitepaper/chapters/cryptography.tex [hunk 3]
```diff
@@ -33,33 +33,33 @@ \subsection{Signing and Verification}
...
+$$SB = rB + (\hf(\underline{R}, \underline{A}, M)a)B = R + \hf(\underline{R}, \underline{A}, M)A$$
 so \eqref{eq:cryptography:verifyR} will hold.
 
 \subsection{Batch Verification}
 
 When lots of signatures have to be verified, a batch signature verification can speed up the process by about 80\%.
 \codenamespace uses the algorithm outlined in \cite{Bernstein2011}.
 Given a batch of $(M_i, A_i, R_i, S_i)$ where $(R_i, S_i)$ is the signature for the message $M_i$ with public key $A_i$,
-uniform distributed 128-bit independent random integers $z_i$ are generated and $H_i(R_i, A_i, M_i)$ is calculated.
+uniform distributed 128-bit independent random integers $z_i$ are generated and $H_i = \hf(R_i, A_i, M_i)$ are calculated.
```
Comment (Jaguar0625) [inline whitepaper/chapters/cryptography.tex]: i think "is" is correct since it's referring to H_i

File: whitepaper/chapters/trees.tex [hunk 4]
```diff
@@ -56,20 +56,20 @@ \subsection{Merkle Tree}
...
+				child { node [highlight] {$H_{CD}$ =\\ $\hf(\hf(C), \hf(D))$ }
+					child { node { $\hf(C)$ } { child { node {C} } } }
+					child { node { $\hf(D)$ } { child { node {D} } } }
 				}
 			}
-			child [sibling distance=16em] { node [highlight] {$H_{EE^2}$ =\\H($H_{EE}$, $H_{EE}$)}
-				child { node {$H_{EE}$ =\\H(H(E), H(E))}
-					child { node {H(E)} { child { node {E} } } }
+			child [sibling distance=16em] { node [highlight] {$H_{EE^2}$ =\\ $\hf(H_{EE}, H_{EE})$}
+				child { node {$H_{EE}$ =\\ }
```
Comment (Jaguar0625) [inline whitepaper/chapters/trees.tex]: did you intentionally remove?

File: whitepaper/chapters/unconfirmed.tex [hunk 5]
```diff
@@ -51,7 +51,7 @@ \subsection{Unconfirmed Transactions Cache}
 Otherwise, it is added back to the cache.
 
 \subsection{Spam Throttle}
-\index{unconfirmedTransactions!spamThrottle}
+\index{unconfirmed transactions!spamThrottle}
```
Comment (Jaguar0625) [inline whitepaper/chapters/unconfirmed.tex]: spam throttle ?

File: whitepaper/preface.tex [hunk 6]
```diff
@@ -21,7 +21,7 @@ \section*{Preface}
 Our mandate was to build a high performance *blockchain* - not a DAG or dBFT based system.
 In this, we think, we succeeded.
 
-This has been a long journey for us, but we are excited to see what yet is to come and what novel things \textit{you} use \codenamespace to build.
+This has been a long journey for us, but we are excited to see what yet is to come and what novel things \emph{you} use \codenamespace to build.
```
Comment (Jaguar0625) [inline whitepaper/preface.tex]: maybe bold this instead ?

File: whitepaper/header/commands.tex [hunk 7]
```diff
@@ -7,13 +7,18 @@
...
-\newcommand{\mathfunc}[1]{\mathrm{#1}}
-\newcommand{\structField}[2]{#1\mathrm{{::}#2}}
+\newcommand{\mathfunc}[1]{\operatorname{#1}}
+\newcommand{\mathvar}[1]{\mathit{#1}}
+\newcommand{\mathname}[1]{\textit{#1}}
+\newcommand{\structField}[2]{%
+	\ifmmode#1{\allowbreak}\mathrm{{::}{\allowbreak}#2}\else\texttt{#1{\allowbreak}::{\allowbreak}#2}\fi
+}
+
+% short hand for hash used in many places
```
Comment (Jaguar0625) [inline whitepaper/header/commands.tex]: short hand => shorthand or alias

File: whitepaper/chapters/blocks.tex (hunk 2 above)
Comment (Jaguar0625) [reply whitepaper/chapters/blocks.tex:75]: this is probably bad idea, and, if desired, can be separate

File: whitepaper/chapters/blockchain_sync_flowchart.tex (hunk 1 above)
Comment (gimre-xymcity) [reply whitepaper/chapters/blockchain_sync_flowchart.tex:2]: yes, to have it both under `block |- synchronization` and separately as `synchronization`
(I assume that is justifiable)

File: whitepaper/chapters/blocks.tex (hunk 2 above)
Comment (gimre-xymcity) [reply whitepaper/chapters/blocks.tex:75]: > i wonder if we should replace all \emph with \nind ?

not sure, some of them are under some existing 'group', i.e. `\index{block!fee multiplier}` and some of them are indexed *earlier*, I'm not sure if it has sense to index *all* occurrences of given term (especially that most of them will be on same or nearby page).

File: whitepaper/chapters/cryptography.tex (hunk 3 above)
Comment (gimre-xymcity) [reply whitepaper/chapters/cryptography.tex]: but there are multiple `H_i`s ? why would there be "are" for `z_i`s and "is" for `H_i`s?

File: whitepaper/chapters/trees.tex (hunk 4 above)
Comment (gimre-xymcity) [reply whitepaper/chapters/trees.tex]: couldn't figure out what you're referring to, now I see it. No, not intentional, fixed.

File: whitepaper/chapters/unconfirmed.tex (hunk 5 above)
Comment (gimre-xymcity) [reply whitepaper/chapters/unconfirmed.tex]: right

File: whitepaper/header/commands.tex (hunk 7 above)
Comment (Jaguar0625) [reply whitepaper/header/commands.tex]: missed this?

File: whitepaper/chapters/cryptography.tex (hunk 3 above)
Comment (Jaguar0625) [reply whitepaper/chapters/cryptography.tex]: Given a batch of $(M_i, A_i, R_i, S_i)$ where $(R_i, S_i)$ is the signature for the message $M_i$ with public key $A_i$, is $H_i = \hf(R_i, A_i, M_i)$ are calculated.
Additionally, uniform distributed 128-bit independent random integers $z_i$ are generated.

File: whitepaper/header/commands.tex (hunk 7 above)
Comment (gimre-xymcity) [reply whitepaper/header/commands.tex]: forgot to commit

File: whitepaper/chapters/cryptography.tex [hunk 8]
```diff
@@ -33,33 +33,34 @@ \subsection{Signing and Verification}
...
 so \eqref{eq:cryptography:verifyR} will hold.
 
 \subsection{Batch Verification}
 
 When lots of signatures have to be verified, a batch signature verification can speed up the process by about 80\%.
 \codenamespace uses the algorithm outlined in \cite{Bernstein2011}.
 Given a batch of $(M_i, A_i, R_i, S_i)$ where $(R_i, S_i)$ is the signature for the message $M_i$ with public key $A_i$,
-uniform distributed 128-bit independent random integers $z_i$ are generated and $H_i(R_i, A_i, M_i)$ is calculated.
+$H_i = \hf(R_i, A_i, M_i)$ is calculated.
+Additionally, corresponding number of uniform distributed 128-bit independent random integers $z_i$ are generated and
```
Comment (Jaguar0625) [inline whitepaper/chapters/cryptography.tex]: ~~and~~
generated. 

?

**Review by Jaguar0625 — APPROVED** [review]

File: whitepaper/chapters/cryptography.tex [hunk 9]
```diff
@@ -33,33 +33,34 @@ \subsection{Signing and Verification}
...
+$$SB = rB + (\hf(\underline{R}, \underline{A}, M)a)B = R + \hf(\underline{R}, \underline{A}, M)A$$
 so \eqref{eq:cryptography:verifyR} will hold.
 
 \subsection{Batch Verification}
 
 When lots of signatures have to be verified, a batch signature verification can speed up the process by about 80\%.
 \codenamespace uses the algorithm outlined in \cite{Bernstein2011}.
 Given a batch of $(M_i, A_i, R_i, S_i)$ where $(R_i, S_i)$ is the signature for the message $M_i$ with public key $A_i$,
-uniform distributed 128-bit independent random integers $z_i$ are generated and $H_i(R_i, A_i, M_i)$ is calculated.
+$H_i = \hf(R_i, A_i, M_i)$ is calculated.
```
Comment (Jaguar0625) [inline whitepaper/chapters/cryptography.tex]: not sure if following is a bit clearer?

Given a batch of $(M_i, A_i, R_i, S_i)$ where $(R_i, S_i)$ is the signature for the message $M_i$ with public key $A_i$, let $H_i = \hf(R_i, A_i, M_i)$.
Additionally, assume a corresponding number of uniform distributed 128-bit independent random integers $z_i$ are generated.

File: whitepaper/chapters/cryptography.tex [hunk 10]
```diff
@@ -33,33 +33,34 @@ \subsection{Signing and Verification}
...
+Additionally, corresponding number of uniform distributed 128-bit independent random integers $z_i$ are generated.
```
Comment (BloodyRookie) [inline whitepaper/chapters/cryptography.tex]: *a* corresponding

File: whitepaper/chapters/reputation.tex [hunk 11]
```diff
@@ -31,7 +31,7 @@ \subsection{Connection Management}
 This guarantees that over time each node will make connections to many different nodes in the network.
 
 \subsection{Weight Based Node Selection}
-\index{reputation!node!selection}
+\index{node!selection}
```
Comment (BloodyRookie) [inline whitepaper/chapters/reputation.tex:34]: why ~~reputation~~?

File: whitepaper/chapters/network.tex [hunk 12]
```diff
@@ -173,11 +173,11 @@ \subsection{Node Discovery}
 \label{sec:network:discovery}
 
 \codenamespace supports a configurable node identification policy configured by \nemsetting{network}{nodeEqualityStrategy}.
```
Comment (BloodyRookie) [inline whitepaper/chapters/network.tex:175]: this exceeds the line ending considerably now, not sure why

File: whitepaper/chapters/reputation.tex (hunk 11 above)
Comment (gimre-xymcity) [reply whitepaper/chapters/reputation.tex:34]: for nicer index (flatten to at most two levels everywhere)

**Review by Jaguar0625 — APPROVED** [review]

**Review by BloodyRookie — APPROVED** [review]

**Comment by gimre-xymcity** [comment]
thx

**Pull request merged**